libs/island-ui/core/src/lib/ProgressMeter/ProgressMeter.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import cn from 'classnames'
+import { Box } from '../Box/Box'
+import * as styles from './ProgressMeter.treat'
+
+export type ProgressMeterVariant = 'blue' | 'red' | 'rose'
+
+type ProgressMeterProps = {
+  /**
+   * Number from 0 to 1
+   */
+  progress: number
+  variant?: ProgressMeterVariant
+  className?: string
+}
+
+const colorSchemes = {
+  blue: {
+    outer: 'blue100',
+    inner: 'blue400',
+  },
+  red: {
+    outer: 'red100',
+    inner: 'red400',
+  },
+  rose: {
+    outer: 'roseTinted100',
+    inner: 'roseTinted400',
+  },
+} as const
```

Comment:
is `as const` necessary here?

Reply:
Yeah, else it's just strings.
`Type 'string' is not assignable to type '"transparent" | "currentColor" | "white" | "dotted"`